### PR TITLE
[DONATOR] [NO GBP] Publicizes MaSvedish's Holocigar

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
image

It shall be done.

image
This publicizes Maxwell Godfrey's holocigar, allowing snails such as this to chuff back a fat dart. Or anyone.

image
(artist's rendition)

image
You will note the presence of the chembaron simply known as 'MG' in the description now.

## How This Contributes To The Skyrat Roleplay Experience
image

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>
  It's in the desc.
</details>

## Changelog
:cl:
add: The holocigar is now public.
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
why does THIS need head input?

**Comment by MaSvedish** [comment]
i can 100% confirm that this is just making a donator item public, and a change to said donator item. I am stupid and don't understand code, this is why I asked [Nerev4r] for help.

**Comment by vinylspiders** [comment]
> why does THIS need head input?

Because @GoldenAlpharex deals with donor items. This label just makes it easier for them to see the PRs that are waiting for them to take a look at.

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -667,7 +667,7 @@
 /datum/loadout_item/pocket_items/masvedishcigar
 	name = "Holocigar"
 	item_path = /obj/item/clothing/mask/holocigarette/masvedishcigar
-	ckeywhitelist = list("masvedish", "lutowski", "lawful", "anyacers", "apolloafk", "avianaviator", "notdhu", "plejek")
+	//ckeywhitelist = list("masvedish", "lutowski", "lawful", "anyacers", "apolloafk", "avianaviator", "notdhu", "plejek") // Asked it to be public.
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
	// Asked it to be public, and as such has no whitelist.
```

**Review by GoldenAlpharex — APPROVED** [review]
Should be good to go!

**Pull request merged**